Fifa/Flask.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello():
+    return "Hello"
+
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
can you also make it run with one of the production grade web servers ?
Here are the most used ones: 
 - gunicorn
 - uwsgi
 - mod_wsgi (apache)

A nice solution is to provide two scripts or configuration arguments by which one can run either the development server or the production server